pylatex/lists.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+"""
+    pylatex.lists
+    ~~~~~~~
+
+    This module implements the class that deals with latex List objects
+    specifically Enumerate and Itemize.
+
+    :copyright: (c) 2015 by Sean McLemon.
+    :license: MIT, see License for more details.
+"""
+
+from .base_classes import BaseLaTeXNamedContainer
+
+
+def Item(s, label=None):
+    u"""Returns the string itemized.
+
+        :param s:
+
+        :type s: str
+
+        :return:
+        :rtype: str
+    """
+    if label:
+        return r'\item[' + label + ']{' + s + '}'
+
+    return r'\item{' + s + '}'
```

Comment:
Why are there curly braces after the label? From my knowledge a space is enough. Your comments in the example file suggest that way as well.